GuessTheNumber.java

```diff
@@ -21,4 +24,129 @@
  * Before the next game, the top 3 scores (which are greater than 0) will be printed
  */
 public class GuessTheNumber {
+
+    private static byte move = 0;
+    private static byte samePosition;
+

```

Comment:
we haven't learned about modifiers yet. static variables should be carefully used - here, 'samePosition' could've been declared as a local variable for example

Reply:
@ameedf i initially declared it as a local variable, but then the main method didn't recognize it.
I don't like declaring global variables at all.

Reply:
if the main method doesn't recognize it (which makes sense) then declare it inside the "main" and pass it to the relevant method when needed. But again - we'll learn more when we get there :)